package.json

```diff
@@ -78,12 +79,16 @@
     "stripe": "9.16.0",
     "tailwindcss": "3.1.7",
     "typescript": "~4.5",
+    "xast-util-to-xml": "3.0.0",
     "xml-js": "1.6.11",
```

Comment:
My reasoning for this is maybe somewhat idiosyncratic, I admit, but it's mostly that I really like the work done around the unified ecosystem and am quite familiar with it.

In case you are unfamiliar, [unified](https://unifiedjs.com/) is a framework/standard/way (don't really know what to call it) for building very modular parsers/converters, mostly focused around markdown to html but with lots of other markup languages supported (there are good ones for org-mode, some experimental ones for LaTeX, and I made a few for JATS and DOCX). 

You use them something like this (packages are not real)
```ts

function markdownToCrossref(markdown: string){
	const mdToCrossref = unified()
								 .use(parseMarkdown)
        	                     .use(markdownToHTML)
            	                 .use(HTMLToXML)
                                 .use(() => (syntaxTree, vfile) => ... // do anything with the syntax tree  
																						// or with the "vfile" which allows you to transfer data in between steps	
										)
                 	             .use(XMLToCrossRef)
                    	         .use(xmlStringify)

	return processor.processSync(markdown)
}
```

What usually bothers me about converters is that they are very ideosyncratic and hard to interop without doing everything from scratch. In unified it's very easy to leverage existing converters as a pathway to your destination, allowing you to skip a lot of the hard work but still giving you a lot of control in the meantime. You can kind of pick how complicated you want to make it, whereas other conversion frameworks are more "all or nothing".

If we use this syntax, I could more easily create e.g. a JATS to CrossRef converter (maybe useless as CrossRef can also accept JATS), but in general I think it would give the library some more longevity as it would be easier to make it work with other conversion tools. This isn't that important for the current implementation, but I'd like to contribute to this ecosystem if I can!

Reply:
Perfect, thanks for explaining! 

That's a great way to move forward and make this much more scalable/extensible. May also prove useful to parse information across the platform :-)